### DISCUSSION TITLE
Is there a way to disable annotation author popup window?

### DISCUSSION THREAD

**Discussion by pulsar123** [sumatrapdfreader/sumatrapdf#3292] [Q&A] [answered]
Thanks again for your great program - the best out there!

I am trying to use it for my podcast, and there is one minor issue: when I hover my mouse over any annotation a popup window appears with the author name. I followed an online advice and set "DefaultAuthor = (none)" in advanced settings, but the windows still pops up with "Unnamed" text. 

Is there a trick to disable this popup window? If not, can you please implement this useful feature (when one sets the author to "(none)" )?

Thank you in advance!

**Comment by GitHubRulesOK** [comment]
Hmmm 
Unnamed is your nom de plume :-) and the whole point of tooltips Is they show the annotation comment, but SumatraPDF 
@kjk It does seem to show Author rather than No Comment? If you use space as a comment nothing should be noticed.
![image](https://user-images.githubusercontent.com/12754913/220465759-85821a32-8b82-4b3c-9e39-ef96ed320639.png) ![image](https://user-images.githubusercontent.com/12754913/220466312-db0ca621-3ea5-4daf-b140-4f1cfdf3bfa6.png)![image](https://user-images.githubusercontent.com/12754913/220467015-40f0712b-6b02-4cb1-a86d-05f29e63ffd6.png)


in some other viewers only author or comments are shown on the tooltip  but not when they are anonymouse
![image](https://user-images.githubusercontent.com/12754913/220467942-65a7e909-80dc-4fe8-84d8-3ebdcf54bca7.png)![image](https://user-images.githubusercontent.com/12754913/220467818-06b13ebc-494b-4da2-b4c4-3336fd916f85.png)![image](https://user-images.githubusercontent.com/12754913/220468058-5def1b29-fdd0-46d0-91e2-3eef336d389a.png)

**Reply by pulsar123** [reply]
Thanks for the tip - using a space as a comment is already an improvement, though it still results in a small (empty) tooltip popping up, plus one has to type the space character for every annotation made. The ideal fix would be an ability to disable annotation tooltips altogether.

**Comment by pulsar123** [comment, marked as answer]
I think I found a solution for my problem: using a non-existent ASCII character as the value for "DefaultAuthor=" string in the Advanced settings. Just go to the Advanced settings (it will open in notepad text editor), point the cursor after the equal sign in the line "DefaultAuthor=", then press Alt key and while holding it type the ascii code 0129 (using the numeric keypad). Once you release Alt, the invisible code is there, even though you won't see anything. Then save the settings (Ctrl-S) and voila - next time you create an annotation, there will be no tooltip window at all (not even an empty one). Problem solved!

This trick can potentially break things in SumatraPDF - use it at your own risk.

**Comment by meteorquake** [comment]
I wonder is there a "formal" solution to this issue - or can one be made?
Popups makes sense in a reviewing situation, but most of the time it's not going to be one, so such author popups are very unwanted, not just for oneself in particular but when doing something and you share it you ideally don't want any popups for the recipient either. If one uses the strange character trick, how can it be gauged what will happen to other users of other software? and then what about PDFs created by other people you don't want the popups on when viewing oneself?
At the very least a boolean option not to have Author popups for oneself would definitely be very very good, and if possible, some way of not writing the authorship so that other people with other software might not see it would be a definite bonus!
Cheers! David

**Reply by GitHubRulesOK** [reply]
@meteorquake 
Tooltips are a Wooly "feature" of the ISO standard for PDFs.
There is no prescribed methods apart from they should be presented for accessibility ! Hence the variation in PDF readers.

The value can be set to  `DefaultAuthor = (none)` when writing but for reading should follow the "standard" but agree "Unnamed" is a fair description feedback for (none)

The "Trick" is using a character that is less visible so even a full stop would work but be just as useless to a blind person.
